From b85847bacb34bae302a7dc1cfcedf8762caefab5 Mon Sep 17 00:00:00 2001
From: shangxiudong <shangxiudong@uniontech.com>
Date: Tue, 2 Jun 2020 09:52:35 +0800
Subject: [PATCH 11/29] Type:fix:23661

Description:When inputting Chinese, right click on the input method icon to switch to Traditional Chinese, restart the input method, the input method Simplified and Traditional status display error

RootCause:Null

Solution:when swich input method, there is a phrase in code which cannot work, I correct it, and let the chttrans'UI update properly
---
 src/module/chttrans/chttrans.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/module/chttrans/chttrans.c b/src/module/chttrans/chttrans.c
index e939972..f49e4f5 100644
--- a/src/module/chttrans/chttrans.c
+++ b/src/module/chttrans/chttrans.c
@@ -238,7 +238,11 @@ void ChttransIMChanged(void* arg)
     FcitxIM* im = FcitxInstanceGetCurrentIM(transState->owner);
     if (!im)
         return;
-    FcitxUIRefreshStatus(transState->owner, "chttrans");
+    boolean enabled = ChttransEnabled(transState);
+    FcitxUISetStatusString(transState->owner, "chttrans",
+                           enabled ? _("Traditional Chinese") :
+                           _("Simplified Chinese"),
+                          _("Toggle Simp/Trad Chinese Conversion"));
 }
 
 /**
-- 
2.20.1

